aries-site/src/components/seo/Meta.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Head from 'next/head';
+
+export const Meta = ({ title, description, canonicalUrl, socialImageUrl }) => {
+  const siteName = 'HPE Aries Design System';
+  const defaultImage = '/static/images/aries-introduction.jp2';
+  const previewImage = socialImageUrl || defaultImage;
+  const twitterHandle = null;
+  const businessUnit = 'CORP';
+  const segment = 'corporate';
+  const lifecycle = 'support';
+  const pageContent = 'products';
+
+  /*
+   * IMPORTANT: `<meta>` tags need to be contained as **direct** children of
+   * the Nextjs `<Head>` component, or wrapped into a maximum one level of
+   * `<React.Fragment>`, otherwise the meta tags will not be correctly picked
+   * up on clientside navigation. For reference: https://nextjs.org/docs#populating-head
+   */
+
+  /*
+   * `key` prop on `<meta>` tags avoids duplicate tags in `<head>` ensuring the
+   * tag is only rendered once. For reference: https://nextjs.org/docs#populating-head
+   */
+
+  return (
+    <Head>
+      <title>{title ? `${title} — Aries` : 'Aries | HPE Design System'}</title>
+      <meta
+        key="viewport"
+        name="viewport"
+        content="width=device-width, initial-scale=1.0"
+      />
+      <meta httpEquiv="Content-Type" content="text/html" charSet="utf-8" />
+      <meta key="description" name="description" content={description} />
+      <link rel="icon" href="/static/favicon.ico" />
+      <link rel="apple-touch-icon" href="/static/favicon.ico" />
+      {canonicalUrl && <link rel="canonical" href={canonicalUrl} />}
+      {/* Open Graph */}
+      <meta
+        key="og:title"
+        name="og:title"
+        property="og:title"
+        content={title}
+      />
+      <meta
+        key="og:description"
+        name="og:description"
+        property="og:description"
+        content={description}
+      />
+      <meta key="og:type" property="og:type" content="website" />
+      <meta key="og:site_name" property="og:site_name" content={siteName} />
+      {previewImage && (
+        <meta key="og:image" property="og:image" content={previewImage} />
+      )}
+      {/* Twitter */}
+      <meta key="twitter:card" name="twitter:card" content="summary" />
+      <meta key="twitter:title" name="twitter:title" content={title} />
+      <meta
+        key="twitter:description"
+        name="twitter:description"
+        content={description}
+      />
+      {twitterHandle && (
```

Comment:
Reading your comment here. So if we combine them, it will be too nested to be picked up? Or putting them in a fragment would be okay here?